build.sbt

```diff
@@ -19,7 +18,7 @@ lazy val buildSettings = Seq(
         <email>dev-role@allenai.org</email>
       </developer>
     </developers>)
-) ++ PublishTo.sonatype
+) ++ bintray.Plugin.bintrayPublishSettings
```

Comment:
I removed this from the bintray plugin as I said I might.  It will be annoying for projects that don't publish to bintray.